command-expansion-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -31,13 +31,13 @@ export type CommandOptions<
   M extends Record<string, any> = Record<string, any>,
 > = { stem: string; arguments: A; metadata?: M } & (
   | {
-      absoluteTime: Temporal.Instant;
+      absoluteTime: String | Temporal.Instant;
```

Comment:
String is not something you should generally use see https://dev.to/josiasaurel/the-difference-between-string-and-string-in-typescript-4mf2